src/main/java/ru/lanwen/diff/uri/UriDiffer.java

```diff
@@ -57,16 +60,18 @@ public UriDiffer expected(URI expected) {
     }
 
 
-    public UriDiffer filter(List<UriDiffFilter> filters) {
-        for (UriDiffFilter filter : filters) {
-            expected(filter.apply(expected));
-            actual(filter.apply(actual));
+    public UriDiffer filter(Collection<UriDiffFilter> filters) {
+        if (filters != null) {
+            for (UriDiffFilter filter : filters) {
+                expected(filter.apply(expected));
+                actual(filter.apply(actual));
+            }
         }
         return this;
     }
 
     public UriDiffer filter(UriDiffFilter... filters) {
-        return filter(asList(filters));
+        return filters != null ? filter(asList(filters)) : this;
```

Comment:
NP, I can add this check.

I just like when API / library is null-safe. It allows to prevent a lot of unnecessary code in the real project.